apple/internal/partials/swift_dylibs.bzl

```diff
@@ -171,11 +181,25 @@ def _swift_dylibs_partial_impl(
                 platform_name,
                 binaries_to_check,
                 output_dir,
+                strip_bitcode,
             )
 
             bundle_files.append((processor.location.framework, None, depset([output_dir])))
 
-            swift_support_file = (platform_name, output_dir)
+            swift_support_output_dir = intermediates.directory(
+                ctx.actions,
+                ctx.label.name,
+                "swiftlibs_for_swiftsupport",
+            )
+            # We're not allowed to modify stdlibs that are used for Swift
+            # Support --- ignoring the bitcode mode here.
+            _swift_dylib_action(
+                ctx,
+                platform_name,
+                binaries_to_check,
+                swift_support_output_dir,
+            )
+            swift_support_file = (platform_name, swift_support_output_dir)
             transitive_swift_support_files.append(swift_support_file)
```

Comment:
can this logic move into the `if` below, since it shouldn't need to be done if we're not packaging swift support?